src/slopes.jl

```diff
@@ -0,0 +1,199 @@
+# Reference : Dietmar Ratz - An Optimized Interval Slope Arithmetic and its Application
+using IntervalArithmetic, ForwardDiff
+import Base: +, -, *, /, ^, sqrt, exp, log, sin, cos, tan, asin, acos, atan
+import IntervalArithmetic: mid, interval
+
+function slope(f::Function, x::Interval, c::Number)
+    try
```

Comment:
I think you should remove the `try` blocks and let the function fail if the function is non-slopable.